index.d.ts

```diff
@@ -0,0 +1,36 @@
+declare module 'sw-toolbox' {
+  interface Cache {
```

Comment:
This doesn't really represent a cache but more like cache options. Maybe `CacheOptions` instead?